### PR TITLE
feat: Update Gardener instructions for open beta

### PR DESCRIPTION
**Pull request by fghaas**
Now that Gardener has moved from the "closed beta" to the "open beta" stage,
update the instructions for how to enable the service.

### CODE REVIEW

File: docs/howto/kubernetes/gardener/create-shoot-cluster.md [hunk 1]
```diff
@@ -6,20 +6,25 @@ that, and how to deploy a sample application on such a cluster.
 
 ## Prerequisites
 
-* To access {{k8s_management_service}} functionality from {{gui}}, you need to enroll in
-  our closed beta. For access to the closed beta, contact our
-  [{{support}}](https://{{support_domain}}/servicedesk).
+* The first time you access the {{k8s_management_service}}
+  functionality from {{gui}}, you need to activate the service. You do
```
Comment (colder-is-better) [inline docs/howto/kubernetes/gardener/create-shoot-cluster.md:10]:
```suggestion
  functionality from the {{gui}}, you need to activate the service. You do
```

**Review by colder-is-better — COMMENTED** [review]
Lookin' good (and I only have a nit).

**Comment by Skipper126** [comment]
Hello! The text update looks great! One thing is that we just made a little change to the gardener description, so that it features our blogpost link. The image here needs to be updated then. It now looks like that:
image

**Comment by fghaas** [comment]
> Hello! The text update looks great! One thing is that we just made a little change to the gardener description, so that it features our blogpost link. The image here needs to be updated then.

It doesn't make a huge difference whether or not we include the reference to the blog post *in the screenshot,* and that reference will go away once the service has emerged from beta. I will leave this as-is.

**Review by Skipper126 — APPROVED** [review]

**Comment by Skipper126** [comment]
Sure, makes sense. Marking as approved. :)

**Review by colder-is-better — DISMISSED** [review]

**Review by Skipper126 — APPROVED** [review]

File: docs/howto/kubernetes/gardener/create-shoot-cluster.md (hunk 1 above)
Comment (fghaas) [reply docs/howto/kubernetes/gardener/create-shoot-cluster.md]: Done. Fixed in another place as well.

**Comment by fghaas** [comment]
Had to update one other screenshot since it has changed in the GUI.

And I had to fix a broken link that's unrelated to the Gardener content (see 31054642f0d7c455928d1cf10f014ad27ef37686).

File: docs/howto/kubernetes/gardener/index.md [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+To use {{k8s_management_service}} in {{brand}}, you first need to *activate* the service. You can conveniently do this via the {{gui}}.
+
+To activate {{k8s_management_service}}, select selecting Containers → [{{k8s_management_service}}](https://{{gui_domain}}/containers/gardener) in the side panel.
+Then, click the _Activate {{k8s_management_service}} Service_ button:
+
+!["Activate {{k8s_management_service}}" dialog in {{gui}}](assets/activate-gardener.png)
+
+You only need to do this once.
+
+
```
Comment (colder-is-better) [inline docs/howto/kubernetes/gardener/index.md:20]: Maybe delete that extra line?

Comment (fghaas) [reply docs/howto/kubernetes/gardener/index.md]: If you insist.

**Pull request merged**